docs/README.md

```diff
@@ -1,24 +1,32 @@
 # uPortal Documentation
 
-## Pages
+**NOTE:**  This area is a work-in-progress.  It is intended to serve as primary
+documentation for uPortal version 5.0 (expected mid-2017) and above.
+Documentation for earlier versions of uPortal is availabe in
+[Confluence](https://wiki.jasig.org).  See *Documentation for Previous Releases*
+below.
 
-*   [Ant Help](antHelp.txt)
-*   [Skinning uPortal](SKINNING_UPORTAL.md)
-*   [Using Angular](USING_ANGULAR.md)
-*   [Supported web browsers](SUPPORTED_BROWSERS.md)
-*   [Configuring using JNDI](configure-using-jndi.md)
+## Sections
+
+* [Ant Help](antHelp.txt)
```

Comment:
Unordered bullets in markdown are often given three spaces so all the bullet point styles line up.

``` markdown
*   example
+   example
-   example
1.  example
99. example
```